website/content/en/docs/faq.md

```diff
@@ -123,10 +123,31 @@ func (r *ReconcileMemcached) Reconcile(request reconcile.Request) (reconcile.Res
 	}
 ```
 
+## When using the "Getting Started" guide, I see deepcopy errors and image build fails. How do I fix this?
+
```

Comment:
Not just valid for Getting started. so should not really be so specific. WDYT? 
could we try to re-phase it?

Reply:
I think you're right, this error is more generic, so we can keep the FAQ generic as well.

Reply:
could we try to shape the following description accordingly as well too?

Reply:
Oh, sure, let me fix it real quick.